service/src/test/java/uk/gov/hmcts/reform/fpl/service/removeorder/GeneratedOrderRemovalActionTest.java

```diff
@@ -66,6 +67,8 @@ void shouldGetRemovedOrderWhenNotFinal() {
             )
         );
 
+        generatedOrder.toBuilder().judgeAndLegalAdvisor(JudgeAndLegalAdvisor.builder().build()).build();
```

Comment:
this is not doing anything. The builder clones the object. Is a requirement for in this action for other orders?